katello_rapidnode.py

```diff
@@ -138,16 +135,25 @@ def child_register(parent, child):
         'rpm -Uvh http://{0}/pub/katello-ca-consumer-latest.noarch.rpm'
         .format(parent)
     )
-    commands.append(
-        'subscription-manager register --username admin --auto-attach --force '
-        '--password {0} --org {1} --environment {2} '
-        .format(
-            CONFIG['credentials']['adminpassword'],
-            CONFIG['mainprefs']['orgname'],
-            CONFIG['mainprefs']['contentview'],
+    if CONFIG.get('mainprefs', 'contentview', fallback=None) is not None:
```

Comment:
I prefer the existing approach which is more readable.  Also in future I may plan to extend the use of `get` by providing fallback values
Ref: https://docs.python.org/2/library/configparser.html#ConfigParser.ConfigParser.get